thresholdkey.go

```diff
@@ -8,27 +8,47 @@ import (
 	"math/big"
 )
 
+// Public key for a threshold Paillier scheme.
```

Comment:
As below, this is typically `ThresholdKey is the public key for…`.

That said, thoughts on renaming this `ThresholdPublicKey`?

Reply:
Good point. We have `PublicKey` and `PrivateKey` for the standard scheme and `ThresholdKey` and `ThresholdPrivateKey` for threshold scheme. It should definitely be `ThresholdPublicKey` and `ThresholdPrivateKey`.